server/js/sockets.js

```diff
@@ -22,6 +22,8 @@ global.statusPayloads = {
       primary: {},
       secondary: {},
     },
+    primary: {},
+    secondary: {},
```

Comment:
If monash-human-power/raspicam#45 does end up being about orchestrator status, you could move all this into an `orchestrator` subcomponent and things would be a lot more consistent :)

Reply:
Do the other components (e.g. sensors) use an `orchestrator` as well? I would like to keep it consistent across all components so it's easier to subscribe to all of them.
Perhaps a `mqtt` subcomponent if we really want to? The inconsistency won't be a problem for the implementation after I refactor it.

Reply:
Sensors don't use an orchestrator, however, BOOST does. If you reckon this will all be cleared up next PR however, then don't worry about it